### PR TITLE
integration with argo

### PR DESCRIPTION
**Pull request by alcorf-mizar**
Add some examples of integration with argo

### CODE REVIEW

File: example/integrations/argo/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# 使用argo workflow集成volcano job
+
+argo 资源模板允许用户创建、删除或更新任何kubernetes资源类型（包括CRD），我们可以使用资源模板来将volcano job集成到argo workflow中，利用argo来为volcano增加job依赖管理和DAG流程控制能力。
```
Comment (hzxuzhonghu) [inline example/integrations/argo/README.md:3]: please translate it into english

Comment (alcorf-mizar) [reply example/integrations/argo/README.md]: Translation completed

**Comment by k82cn** [comment]
That's great!

File: example/integrations/argo/README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      action: create					# can be any kubectl action (e.g. create, delete, apply, patch)
+      # The successCondition and failureCondition are optional expressions.
+      # If failureCondition is true, the step is considered failed.
+      # If successCondition is true, the step is considered successful.
+      # They use kubernetes label selection syntax and can be applied against any field
+      # of the resource (not just labels). Multiple AND conditions can be represented by comma
+      # delimited expressions.
+      # For more details: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+      successCondition: status.succeeded > 0
+      failureCondition: status.failed > 3
```
Comment (hzxuzhonghu) [inline example/integrations/argo/README.md:70]: I doubt if this is right

File: example/integrations/argo/README.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
+# Use Argo Workflow to integrate Volcano Job
+
+The Argo resource template allows users to create, delete, or update any type of Kubernetes resource (including CRDs). We can use the resource template to integrate Volcano Jobs into Argo Workflow, and use Argo to add job dependency management and DAG process control capabilities to volcano.
+
+```markdown
+[argo使用kubernetes resource](https://github.com/argoproj/argo/blob/master/examples/README.md#kubernetes-resources)
```
Comment (k82cn) [inline example/integrations/argo/README.md:6]: Should be in English :)

Comment (alcorf-mizar) [reply example/integrations/argo/README.md]: yes.

Comment (k82cn) [reply example/integrations/argo/README.md]: thanks :)

File: example/integrations/argo/20-job-DAG.yaml [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+          parameters: [{name: message, value: D}, {name: task, value: ddd}]
+  - name: hello-tmpl
+    inputs:
+      parameters:
+      - name: message
+      - name: task
+    resource:
+      action: create
+      successCondition: status.succeeded > 1
+      failureCondition: status.failed > 3
```
Comment (hzxuzhonghu) [inline example/integrations/argo/20-job-DAG.yaml:39]: @alcorf-mizar could you fix this?

File: example/integrations/argo/README.md (hunk 2 above)
Comment (hzxuzhonghu) [reply example/integrations/argo/README.md:74]: Also here

**Pull request merged**